pkg/kubectl/describe.go

```diff
@@ -1577,11 +1578,11 @@ func DescribeEvents(el *api.EventList, w io.Writer) {
 
 // DeploymentDescriber generates information about a deployment.
 type DeploymentDescriber struct {
-	client.Interface
+	clientset release_1_1.Interface
```

Comment:
Sure, I'll make it inlined. 

I used to need both `client.Interface` and `release_1_1.Interface`, so I made them named to avoid collision between method names.